### ISSUE TITLE
Rework lists feature, assign tags to lists instead of individual items.

### ISSUE THREAD

**Issue by KristofStroobants** [hoarder-app/hoarder#765]
### Describe the feature you'd like

Right now, you have to manually add items to a list.
Since we have AI tagging for us, it would make more sense to assign tags to a list.
Then once the AI tags your items, they'll be automatically integrated in your lists as well.

### Describe the benefits this would bring to existing Hoarder users

Way less overhead and manual input. You can go in hoarder to manage your lists once a month. Assign the newly created tags to your lists and you are done.

### Can the goal of this request already be achieved via other means?

Not that i am aware of.

### Have you searched for an existing open/closed issue?

- [X] I have searched for existing issues and none cover my fundamental request

**Comment by MohamedBassem** [comment]
This is planned in https://github.com/hoarder-app/hoarder/issues/65 and will be coming in the next release.
TL;DR lists backed by tag searches.
